charts/k8tz/templates/controller.yaml

```diff
@@ -37,6 +41,29 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
+        {{- if .Values.webhook.certManager.enabled }}
+        - name: {{ .Chart.Name }}-watcher
```

Comment:
the first container is the default container when using `kubectl logs`, i prefer the first container to be k8tz itself. can you please reorder them?